assets/js/base/components/radio-control/_mixin.scss

```diff
@@ -24,27 +23,35 @@
 		top: $gap-small;
 	}
 
+	.wc-block-radio-control__label-group,
+	.wc-block-radio-control__description-group {
+		display: table-row;
+
+		span {
+			display: table-cell;
+		}
+
+		.wc-block-radio-control__secondary-label,
+		.wc-block-radio-control__secondary-description {
+			text-align: right;
+			min-width: 50%;
```

Comment:
Would it work removing `min-width` from here? For me, it seems to cause the shipping name to break earlier than necessary:

_with `min-width`:_
![imatge](https://user-images.githubusercontent.com/3616980/81662014-41700e00-943d-11ea-97ae-6e005811bc59.png)

_without `min-width`:_
![imatge](https://user-images.githubusercontent.com/3616980/81661981-387f3c80-943d-11ea-9a3e-0e6634c740ec.png)

Reply:
It's just spans within spans, easy fix :)